cli/command/system/info_test.go

```diff
@@ -279,6 +282,16 @@ func TestPrettyPrintInfo(t *testing.T) {
 			jsonGolden:    "docker-info-errors",
 			expectedError: "errors pretty printing info",
 		},
+		{
+			doc: "bad security info",
+			dockerInfo: info{
+				Info:       &sampleInfoBadSecurity,
```

Comment:
Perhaps the test should have something to return multiple errors (and a warning?)

Reply:
There's only one unit testable error path out of `prettyPrintDaemonInfo` (the security parsing stuff), the other error path is a failure to talk to the daemon which can't be unit tested. I'll add a hardcoded error in the dummy info, that should at least test the appending bit.

Warnings are already covered by pre-existing tests.